.circleci/config.yml

```diff
@@ -284,6 +304,9 @@ workflows:
   test_all:
     jobs:
       - checkout_code
+      - migrations_test:
```

Comment:
I know it's not something we already do, but it could be nice to run this only when at least one file is changed in the directory `src/api/db`.